nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/FlowController.java

```diff
@@ -16,7 +16,38 @@
  */
 package org.apache.nifi.controller;
 
-import com.sun.jersey.api.client.ClientHandlerException;
```

Comment:
@joewitt this change was due to a bug that I ran into when testing this PR. The bug was preventing me from moving forward, and was a rather trivial fix, so I just included it in the PR. I could have created a separate JIRA and PR but since it would have blocked this PR it seemed more prudent to just roll it in.